src/current/v23.1/ui-hot-ranges-page.md

```diff
@@ -34,7 +34,12 @@ To view the [Range Report](#range-report) for a hot range, click its range ID.
 Parameter | Description
 ----------|------------
 Range ID | The ID of the hot range. Click the range ID to view the [Range Report](#range-report) for this range.
-QPS | Total number of `SELECT`, `UPDATE`, `INSERT`, and `DELETE` queries executed per second on this range.
+QPS | Total number of `SELECT`, `UPDATE`, `INSERT`, and `DELETE` queries executed per second on this range. The per-second rate is averaged over the last 30 minutes.
```

Comment:
nit: For consistency, this description should start with "The..."
You might want to also link out to the statements' pages

Reply:
Done